components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml

```diff
@@ -950,6 +952,12 @@ paths:
             Comments list is returned.
           schema:
             $ref: '#/definitions/CommentList'
+        401:
+          description: |
+            Un authorized.
+            The user is not authorized to view the application .
```

Comment:
Un authorized =>  Unauthorized
Shall we fix the description as well?

Reply:
fixed

Reply:
"The user is not authorized to view the application:" description may need to be fixed